demos/text_detection_demo/cpp/README.md

```diff
@@ -6,6 +6,7 @@ The demo shows an example of using neural networks to detect and recognize print
 * `text-detection-0004`, which is a lightweight detection network for finding text.
 * `horizontal-text-detection-0001`, which is a detection network that works much faster than models above, but it is applicable to finding more or less horizontal text only.
 * `text-recognition-0012`, which is a recognition network for recognizing text.
+* `text-recognition-0013`, which is a recognition network for recognizing text.
```

Comment:
is it added as replacement for 0012 or we will have 2 models?

Reply:
For now, this would be an additiononal model.